wdl-doc/src/meta.rs

```diff
@@ -1,60 +1,289 @@
+            Markdown(inner_text).render()
         }
-        MetadataValue::Boolean(b) => html! { code { (b.text().to_string()) } },
-        MetadataValue::Integer(i) => html! { code { (i.text().to_string()) } },
-        MetadataValue::Float(f) => html! { code { (f.text().to_string()) } },
-        MetadataValue::Null(n) => html! { code { (n.text().to_string()) } },
+        MetadataValue::Boolean(b) => html! { code { (b.text()) } },
+        MetadataValue::Integer(i) => html! { code { (i.text()) } },
+        MetadataValue::Float(f) => html! { code { (f.text()) } },
+        MetadataValue::Null(n) => html! { code { (n.text()) } },
         MetadataValue::Array(a) => {
             html! {
-                div {
-                    code { "[" }
-                    ul {
-                        @for item in a.elements() {
-                            li {
-                                @match item {
-                                    MetadataValue::Array(_) | MetadataValue::Object(_) => {
-                                        (render_value(&item)) ","
-                                    }
-                                    _ => {
-                                        code { (item.text().to_string()) } ","
-                                    }
+                div class="main__grid-meta-array-container" {
+                    @for item in a.elements() {
+                        @match item {
+                            MetadataValue::Array(_) | MetadataValue::Object(_) => {
+                                // This is going to render weirdly. I (a-frantz)
+                                // don't have a real example case for this,
+                                // so I'm leaving it as is for now. This would be a very
+                                // odd structure in WDL metadata, but it is valid.
+                                (render_value_inner(&item))
+                            }
+                            _ => {
+                                div class="main__grid-meta-array-item" {
+                                    code { (item.text()) }
                                 }
                             }
                         }
                     }
-                    code { "]" }
                 }
             }
         }
         MetadataValue::Object(o) => {
             html! {
-                div {
-                    code { "{" }
-                    ul {
-                        @for item in o.items() {
-                            li {
-                                b { (item.name().text()) ":" } " " (render_value(&item.value())) ","
+                div class="main__grid-nested-container" {
+                    @for item in o.items() {
+                        (render_key_value(item.name().text(), &item.value()))
+                    }
+                }
+            }
+        }
+    }
+}
+
+/// Render a [`MetadataValue`] as HTML.
+fn render_value(value: &MetadataValue) -> Markup {
```

Comment:
Is this really needed, or can we delete it and just rename `render_value_inner` as `render_value`?

Reply:
No, it's an artifact of when there was additional logic in `render_value()` that wasn't present in the recursive call, but that logic was removed at some point and the functions weren't consolidated. Will add it to the batch of fixes 👍